custom_components/hubitat/switch.py

```diff
@@ -128,7 +128,10 @@ def is_switch(device: Device, overrides: Optional[Dict[str, str]] = None) -> boo
 
 def is_energy_meter(device: Device, overrides: Optional[Dict[str, str]] = None) -> bool:
     """Return True if device can measure power."""
-    return DeviceCapability.POWER_METER in device.capabilities
+    return (
+        DeviceCapability.POWER_METER in device.capabilities
+        or DeviceCapability.ENERGY_METER in device.capabilities
```

Comment:
Does including ENERGY_METER here gain anything? `is_energy_meter` is just use to determine wheterh to use a HubitatPowerMeterSwitch or a regular HubitatSwitch, and HubitatPowerMeterSwitch only looks at the `power` attribute, not `energy`.